src/components/HomePage/MainView.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import logo from './nijobs.png';
+
+import SearchArea from "./SearchArea/SearchArea";
+import ShowMoreButton from "./ShowMoreButton";
+import InfoBox from "./QuickInfoArea/InfoBox";
+
+import homePageStyles from './HomePage.module.css';
+
+const MainView = props => {
+
+    const {
+        scrollToProductDescription, 
+        scrollToSearchResults
+    } = props;
```

Comment:
Use object destructuring for `props`

Reply:
I thought about it when making this, but I figured it would be more readable if it was like this, in case the props increase.

Reply:
Unlikely imo, but can understand the idea. I personally prefer to go with the flow, as a little bit of future proofing can easily become too much 😅 Thus, would still vouch for destructuring 😄